### PR TITLE
Process long CLI args properly

### PR DESCRIPTION
**Pull request by tgetgood**
There was an error in the command line arg processing where long options (`--after`, etc.) were being dropped. This should fix it.

Addresses #14

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -34,27 +34,33 @@ const cli = meow([`
   }
 }])
 
-const token = cli.flags.t || process.env.GITHUB_TOKEN
+const user = cli.flags.u || cli.flags.user
```
Comment (RichardLitt) [inline src/cli.js]: This is the point of the alias section above. I think we're doing something wrong if this is needed, too?

Comment (tgetgood) [reply src/cli.js]: You are correct. Unfortunately meow isn't clearly documented (the examples don't match the reference) and reading the source code, the alias is never referred to.

In fact the library doesn't appear to provide us with anything except reading `process.argv` and a gif of a kitten puking rainbows. Undoubtedly cute, but not very useful.

So short of writing our own I think this is the best we get.

Comment (RichardLitt) [reply src/cli.js]: Can we just use minimist instead, then?

Comment (RichardLitt) [reply src/cli.js]: Looking at meow, it should get minimist opts fine. https://www.npmjs.com/package/minimist

**Comment by RichardLitt** [comment]
Closed by 992a0c4. Ran into this problem before, should've spotted it.

**Pull request closed without merging**